test/examples/transactions.js

```diff
@@ -221,4 +222,69 @@ describe('examples(transactions):', function() {
       // End Transactions Retry Example 3
     }
   });
+
+  it('Transactions withTransaction API Example 1', {
+    metadata: { requires: { topology: ['replicaset'], mongodb: '>=3.8.0' } },
+    test: async function() {
+      const uri = this.configuration.url();
+
+      // Start Transactions withTxn API Example 1
+
+      // For a replica set, include the replica set name and a seedlist of the members in the URI string; e.g.
+      // const uri = 'mongodb://mongodb0.example.com:27017,mongodb1.example.com:27017/?replicaSet=myRepl'
+      // For a sharded cluster, connect to the mongos instances; e.g.
+      // const uri = 'mongodb://mongos0.example.com:27017,mongos1.example.com:27017/'
+
+      const client = new MongoClient(uri);
+      await client.connect();
+
+      // Prereq: Create collections. CRUD operations in transactions must be on existing collections.
+
+      client
```

Comment:
don't you need to await these two operations?

Reply:
yes, will update shortly